infra/client/product-planner-ng/src/app/categories/categories.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { EditCategoriesComponent } from './edit-categories/edit-categories.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/maciejmalecki/inventory/pullRequest?prid=7113829)
```suggestion
import { EditCategoriesComponent } from "./edit-categories/edit-categories.component";
```